keyboards/varanidae/keymaps/via/rules.mk

```
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

To make the encoder rotation configurable in VIA, you need to enable the encoder map support:
```suggestion
VIA_ENABLE = yes
ENCODER_MAP_ENABLE = yes
```
(and also include the encoder map in `keymap.c`, otherwise the code won't compile).

Actually you can do the same changes in the `default` keymap too, so that the way to configure the encoder would be obvious to some user who decides to copy the `default` keymap before doing their modifications.